server/utils/utils.ts

```diff
@@ -54,6 +62,119 @@ export function formatDate(date: string): string {
   })
 }
 
+export const formatAssessmentData = (
+  crimNeeds: CriminogenicNeedsData,
+  assessment: AssessmentResponse,
+  areas: AssessmentAreaConfig[],
+  forename: string,
+): AssessmentAreas => {
+  if (assessment === null) {
+    return { lowScoring: [], highScoring: [] }
+  }
+  const all = Object.values(areas)
+    .map(area => {
+      let score
+      let linkedtoRoSH
+      let linkedtoReoffending
+
+      if (Object.prototype.hasOwnProperty.call(crimNeeds, area.crimNeedsKey)) {
+        score = crimNeeds[area.crimNeedsKey][`${area.crimNeedsSubKey}OtherWeightedScore`]
+        linkedtoRoSH = crimNeeds[area.crimNeedsKey][`${area.crimNeedsSubKey}LinkedToHarm`] === 'YES'
+        linkedtoReoffending = crimNeeds[area.crimNeedsKey][`${area.crimNeedsSubKey}LinkedToReoffending`] === 'YES'
+
+        if (Number.isNaN(Number(score))) {
+          score = undefined
+        }
+      }
+
+      const motivationToMakeChanges = motivationText(
+        assessment.assessment[`${area.assessmentKey}_changes`]?.value,
+        forename,
+      )
+      const riskOfSeriousHarm =
+        assessment.assessment[`${area.assessmentKey}_practitioner_analysis_risk_of_serious_harm_yes_details`]?.value
+      const riskOfReoffending =
+        assessment.assessment[`${area.assessmentKey}_practitioner_analysis_risk_of_reoffending_yes_details`]?.value
+      const strengthsOrProtectiveFactors =
+        assessment.assessment[`${area.assessmentKey}_practitioner_analysis_strengths_or_protective_factors_yes_details`]
+          ?.value
+
+      return {
+        title: area.area,
+        linkedtoRoSH,
+        linkedtoReoffending,
+        motivationToMakeChanges,
+        riskOfSeriousHarm,
+        riskOfReoffending,
+        strengthsOrProtectiveFactors,
+        criminogenicNeedsScore: score,
+        goalRoute: area.goalRoute,
+      } as AssessmentArea
+    })
+    .sort((a, b) => 0 - (a.criminogenicNeedsScore > b.criminogenicNeedsScore ? 1 : -1))
+
+  const lowScoring = all.filter(area => Number(area.criminogenicNeedsScore) <= 3)
+  const highScoring = all.filter(area => Number(area.criminogenicNeedsScore) > 3)
+  return { lowScoring, highScoring, versionUpdatedAt: assessment.metaData?.versionUpdatedAt } as AssessmentAreas
+}
+
+export const toHtml = (content: string): string => {
+  return content.replace('\n', '&#13;&#10;')
+}
+
+export const motivationText = (optionResult: string, forename: string): string => {
```

Comment:
Is this something we could do via the locale?

Reply:
Removed function above as thats unused. Looking at using locales

Reply:
Now using locales